src/transport/websocket.ts

```diff
@@ -257,6 +255,18 @@ export default class WebSocketTransport implements SubscriptionTransport {
       return;
     }
 
+    // In Chrome there is a substantial delay between calling close on a broken
+    // websocket and the onclose method firing. When we're force closing the
+    // connection we can expedite the reconnect process by manually calling
+    // onclose. We then need to delete the socket's handlers so that we don't
+    // get extra calls from the dying socket.
+    const onClose = this.socket.onclose.bind(this);
```

Comment:
I'm confused as to why we bind to `this`.

Reply:
I mean, I know why we want the code to be executed in the context of our `this`, it's declared as an anonymous function further up in this class, but I'm surprised that we need to bind it here but not there.